packages/yourloops/components/device/generic-list-card.tsx

```diff
@@ -25,40 +25,50 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import styles from '../diabeloop.css'
-import React, { type FunctionComponent } from 'react'
-import { type HistorizedParameter } from '../../../models/historized-parameter.model'
-import type moment from 'moment-timezone'
+import React, { type FC, type PropsWithChildren } from 'react'
+import Card from '@mui/material/Card'
+import CardHeader from '@mui/material/CardHeader'
+import CardContent from '@mui/material/CardContent'
+import List from '@mui/material/List'
+import Divider from '@mui/material/Divider'
+import { type Theme, useTheme } from '@mui/material/styles'
+import { makeStyles } from 'tss-react/mui'
 
-interface HistorySpannedRowProps {
-  data: HistorizedParameter
-  onSwitchToDaily: (date: moment.Moment | Date | number | null) => void
+interface GenericListCardProps extends PropsWithChildren {
+  title: string
+  ['data-testid']?: string
 }
 
-const ONE_SPACE_STRING = '&nbsp;'
-const NUMBER_OF_COLUMNS = 4
+const useStyles = makeStyles()((theme: Theme) => ({
+  cardHeader: {
+    backgroundColor: 'var(--primary-color-background)',
+    fontSize: theme.typography.fontSize,
+    fontWeight: theme.typography.fontWeightBold
+  }
+}))
 
-export const HistorySpannedRow: FunctionComponent<HistorySpannedRowProps> = (props) => {
-  const { onSwitchToDaily, data } = props
-  const content = data.groupedParameterHeaderContent ?? ONE_SPACE_STRING
+export const GenericListCard: FC<GenericListCardProps> = (props) => {
+  const theme = useTheme()
+  const { classes } = useStyles()
+  const { title, children } = props
 
-  const handleSwitchToDaily = (): void => {
-    onSwitchToDaily(data.latestDate)
-  }
-  const dateString = data.latestDate.toString()
   return (
-    <tr className={styles.spannedRow} >
-      <td colSpan={NUMBER_OF_COLUMNS}>
-        {content}
-        <i
-          role="button"
-          data-testid={`button-${content}`}
-          tabIndex={0}
-          data-date={dateString}
-          className={`icon-chart-line ${styles.clickableIcon}`}
-          onClick={handleSwitchToDaily}
-        />
-      </td>
-    </tr>
+    <Card
+      variant="outlined"
+      sx={{ marginBottom: theme.spacing(5) }}
+      data-testid={props['data-testid']}
+    >
+      <CardHeader
+        title={title}
+        className={classes.cardHeader}
+        disableTypography
+      />
+      <CardContent sx={{ padding: '0 !important', fontSize: theme.typography.fontSize }}>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is it alright to use `!important`?

Reply:
removing it in favor of a class ;)